Testreglar/253/App/app-2.5.3a.json

```diff
@@ -0,0 +1,153 @@
+{
+	"id": "app-2.5.3a",
+	"type": "app",
+	"kravTilSamsvar": "<p>For brukergrensesnittkomponenter med ledetekster som inneholder tekst eller bilder av tekst, som støtter tilgjengelig navn, gjelder en av følgende:</p>\r\n<ul>\r\n<li>Synlig (visuell) ledetekst og tilgjengelig navn (accessible name) er identiske eller</li>\r\n<li>Tilgjengelig navn (accessible name) inneholder samme tekst, i samme rekkefølge som synlig (visuell) ledetekst </li>\r\n</ul>",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken appside tester du?",
+			"ht": "<p>Oppgi appside-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsiden interaktive brukergrensesnittkomponenter? ",
+			"ht": "<p>Gjøre visuell inspeksjon og identifiser interaktive brukergrensesnittkomponenter. Dette gjelder blant annet brukergrensesnittkomponenter med rolle <a href=\"https://www.w3.org/TR/wai-aria-1.1/#widget_roles\" target=\"_blank\" rel=\"noopener\">widget</a> eller tilsvarende for apper (<a href=\"https://developer.apple.com/design/human-interface-guidelines/components\" target=\"_blank\" rel=\"noopener\">iOS</a> og <a href=\"https://m3.material.io/components\" target=\"_blank\" rel=\"noopener\">Android</a>).</p>\n<p><strong>Merk: </strong>Skjema i PDF, Word eller liknende, er ikke aktuelle testobjekt.</p>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "2.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Appsiden har ikke interaktive brukergrensesnittkomponenter."
+				}
+			}
+		},
+		{
+			"stegnr": "2.3",
+			"spm": "Hvilken type brukergrensesnittkomponenten er det?",
+			"ht": "<p>Velg fra alternativ:</p>",
+			"type": "radio",
+			"kilde": [],
+			"svarArray": [
```

Comment:
samme kommentar som for nett. Endre til flervalg? Eller flytte til steg etter registrering av element?

Reply:
Dersom dette skal registreres må det flyttes til etter 3.1 siden dette er en egenskap med hver enkelt **brukergrensesnittkomponent**